docs/reference/transform/transform-alerts.asciidoc

```diff
@@ -0,0 +1,87 @@
+[role="xpack"]
+[[transform-alerts]]
+= {transform-cap} alerts
+
+beta::[]
+
+{kib} {alert-features} include support for {transform} rules, which check the 
+health of {ctransforms} with certain conditions. If the conditions of the rule 
+are met, an alert is created and the associated action is triggered. For 
+example, you can create a rule to check if a {ctransform} is started and to 
+notify you in an email if it is not. To learn more about {kib} {alert-features}, 
+refer to 
+{kibana-ref}/alerting-getting-started.html#alerting-getting-started[Alerting].
+
+The following {transform} rules are available:
+
+{transform-cap} health:: 
+  Monitors {transforms} health and alerts if an operational issue occurred.
+
+
+[[creating-transform-rules]]
+== Creating a rule
+
+You can create {transform} rules under **{stack-manage-app} > {alerts-ui}**.
```

Comment:
I see "Alerts and Actions" in the preview but it should be "Rules and Connectors"

Reply:
I noticed a similar issue with the last paragraph about defining actions

Reply:
```suggestion
You can create {transform} rules under **{stack-manage-app} > {rules-ui}**.
```

Reply:
Addressed in aa0b52d.